src/client/src/orangehrmLeavePlugin/pages/LeaveRequest.vue

```diff
@@ -260,7 +262,9 @@ export default {
       if (employee) {
         const name = `${employee.firstName} ${employee.middleName}
         ${employee.lastName}`;
-        return `${name} ${employee.terminationId ? '(Past Employee)' : ''}`;
+        return `${name} ${
```

Comment:
For Transforming employee object to full name/display name with past employee tag use [useEmployeeNameTranslate](https://github.com/orangehrm/orangehrm/blob/main/src/client/src/core/util/composable/useEmployeeNameTranslate.ts)

```
...
setup() {
    const {$tEmpName} = useEmployeeNameTranslate();

    return {
       // vue don't allow variables prefixed with $ sign to return from setup
      translateEmpName: $tEmpName,
    };
},

computed: {
    employeeName() {
      const employee = this.response?.meta?.employee;
      if (employee) {
        return this.translateEmpName(employee, {
            includeMiddle: true,
            excludePastEmpTag: false,
        });
      }
      return '';
    }
}
```

[ref](https://github.com/orangehrm/orangehrm/blob/faf46890b5b1aa56bf1756d9fbea2118e372b971/src/client/src/orangehrmRecruitmentPlugin/pages/ViewActionHistory.vue#L256)

Reply:
seems not done with `useEmployeeNameTranslate`